src/main/java/qna/domain/Answer.java

```diff
@@ -99,6 +101,19 @@ public void setDeleted(boolean deleted) {
         this.deleted = deleted;
     }
 
+    public DeleteHistory delete(User loginUser, LocalDateTime deletedAt) {
+        if (!isOwner(loginUser))
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+
```

Comment:
```suggestion
        if (!isOwner(loginUser)) {
            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
        }
```

자바에서는 한라인도 괄호로 감싸주는 것이 관례입니다 :)

Reply:
넵 ! 또 하나 배워갑니다 😊